app/scripts/controllers/ModalCtrl.js

```diff
@@ -1,18 +1,17 @@
 (function() {
-  //Do I need to include $log and $document as dependents?
-  function ModalCtrl($uibModal, $log, $document, Room) {
+  function ModalCtrl(Room, $uibModal) {
```

Comment:
Actually not first, so something must have been off with `$log` or `$document` that was causing it to break the injection.